packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -54,6 +54,10 @@ export interface IDPointType {
   values: { 0: number; 1: number; data: {} };
   xVal: number | Date;
 }
+export interface IMapXToDataSet {
+  [key: string]: ILineChartDataPoint[];
+  [key: number]: ILineChartDataPoint[];
```

Comment:
where is the code to limit point precision?

Reply:
Lets add this as a comment in our code.

Reply:
Done